protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1613,6 +1620,18 @@ message TransferConnection {
   PageInfo page_info = 2;
 }
 
+// Request that is sent for getting a transfer by ID
+message GetTransferByIDRequest {
+  // Transfer ID to request data for.
+  string transfer_id = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+// Response that is received when querying transfers by ID
+message GetTransferByIDResponse {
+  // page containing the transfer requested by ID.
```

Comment:
should this be a capital 'P' for consistency as per the other similar strings above

Reply:
This is actually an incorrect comment, must've deleted the wrong line